src/components/Parallax/ParallaxComponent.js

```diff
@@ -2,24 +2,36 @@ import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 
 function ParallaxComponent(props) {
-  const { children, shouldParallaxScroll, transitionTime, transitionDelay } = props;
+  const { children, shouldParallaxScroll, transitionTime, transitionDelay, resetTime } = props;
 
   const parallaxContainer = useRef(null);
 
+  const disableAnimation = () => {
+    parallaxContainer.current.style.transition = `transform 0ms ease-in-out`;
+    parallaxContainer.current.style.transitionDelay = `0ms`;
+  };
+
   const slideDown = () => {
     parallaxContainer.current.style.transform = 'translate(0,200px)';
   };
 
+  const enableAnimation = () => {
+    parallaxContainer.current.style.transition = `transform ${transitionTime}ms ease-in-out`;
+    parallaxContainer.current.style.transitionDelay = `${transitionDelay}ms`;
+  };
+
   useEffect(() => {
     parallaxContainer.current.style.transform = 'translate(0,0)';
     if (!shouldParallaxScroll) {
-      const timeout = setTimeout(slideDown, transitionTime);
+      disableAnimation();
+      const timeout = setTimeout(slideDown, resetTime);
+      setTimeout(enableAnimation, resetTime + 10);
       return () => {
         clearTimeout(timeout);
       };
     }
     return () => {};
-  }, [shouldParallaxScroll, transitionTime]);
+  }, [shouldParallaxScroll, resetTime]);
```

Comment:
I'm getting an compile warning saying enableAnimation should be a dependency in the dependency array. However when you add it says it would cause the effect to refresh on every update, defeating the purpose of a dependency array. I'm gonna leave the warning in for now, since you may be refactoring this code anyways.